src/Tribe/Admin/Help_Page.php

```diff
@@ -0,0 +1,451 @@
+<?php
+
+// Don't load directly
+if ( ! defined( 'ABSPATH' ) ) {
+	die( '-1' );
+}
+
+/**
+ * Class with a few helpers for the Administration Pages
+ */
+class Tribe__Admin__Help_Page {
+	/**
+	 * Static Singleton Holder
+	 * @var Tribe__Admin__Helpers|null
+	 */
+	protected static $instance;
+
+	/**
+	 * Static Singleton Factory Method
+	 *
+	 * @return Tribe__Admin__Helpers
+	 */
+	public static function instance() {
+		if ( ! isset( self::$instance ) ) {
+			$className      = __CLASS__;
+			self::$instance = new $className;
+		}
+
+		return self::$instance;
+	}
+
+	/**
+	 * Get the list of plugins
+	 *
+	 * @param  string  $plugin_name    Should get only one plugin?
+	 * @param  boolean $is_active Only get active plugins?
+	 * @return array
+	 */
+	public function get_plugins( $plugin_name = null, $is_active = true ) {
+		$plugins = array();
+
+		$plugins['the-events-calendar'] = array(
+			'name' => 'the-events-calendar',
+			'title' => __( 'The Events Calendar', 'tribe-common' ),
+			'api' => 'http://wpapi.org/api/plugin/the-events-calendar.php',
+			'repo' => 'http://wordpress.org/extend/plugins/the-events-calendar/',
+			'stars_url' => 'http://wordpress.org/support/view/plugin-reviews/the-events-calendar?filter=5',
+			'description' => __( 'The Events Calendar is a carefully crafted, extensible plugin that lets you easily share your events.', 'tribe-common' ),
+			'is_active' => false,
+			'version' => null,
+		);
+
+		if ( class_exists( 'Tribe__Events__Main' ) ){
```

Comment:
Minor nitpick (rinse and repeat for other if statements): `) {` vs `){`

We really need to get a rule in place to sniff this.